packages/services/storage/src/index.ts

```diff
@@ -519,6 +520,10 @@ export async function createStorage(
                     WHERE organization_id = ${linkedOrganizationId} AND name = 'Viewer')
                 )
               ),
+              (
+                SELECT default_assigned_resources FROM oidc_integrations
+                WHERE id = ${args.oidcIntegrationId}
```

Comment:
should this check organizationId also for extra confidence that the oidc belongs to the correct org?